### PR TITLE
Fixes for TRAFODION-1789 & TRAFODION-1812

### PR DESCRIPTION
**Pull request by robertamarton**
TRAFODION-1789: Create the histogram metadata tables at schema creation time
TRAFODION-1812: cleanup schema causes transaction to hang in aborting state

### CODE REVIEW

File: core/sql/bin/SqlciErrors.txt [hunk 1]
```diff
@@ -72,7 +72,7 @@
 1070 ZZZZZ 99999 BEGINNER MAJOR DBADMIN Object $0~TableName could not be created. File error: $1~Int0.
 1071 ZZZZZ 99999 BEGINNER MAJOR DBADMIN Object $0~TableName could not be accessed. 
 1072 ZZZZZ 99999 BEGINNER MAJOR DBADMIN Unique constraint $0~String0 is disabled, so foreign key constraint $1~ConstraintName could not be created.
-1073 ZZZZZ 99999 BEGINNER MAJOR DBADMIN Only services ID can execute INITIALIZE SQL.
+1073 ZZZZZ 99999 BEGINNER MAJOR DBADMIN Schema $0~SchemaName was partially dropped, call CLEANUP SCHEMA to remove remaining entries.
```
Comment (DaveBirdsall) [inline core/sql/bin/SqlciErrors.txt]: Suggested wordsmith: "Schema $0~SchemaName was only partially dropped due to errors. Use CLEANUP SCHEMA to complete."

File: core/sql/sqlcomp/CmpSeabaseDDLschema.cpp [hunk 2]
```diff
@@ -458,132 +479,159 @@ ComObjectName objName(catName,schName,NAString("dummy"),COM_TABLE_NAME,TRUE);
...
-      return;
+      goto label_error;
    }
 
-   objectsQueue->position();
-   if ((dropSchemaNode->getDropBehavior() == COM_RESTRICT_DROP_BEHAVIOR) &&
-       (objectsQueue->numEntries() > 0))
+   // Check to see if non histogram objects exist in schema, if so, then 
+   // cascade is required
+   if (dropSchemaNode->getDropBehavior() == COM_RESTRICT_DROP_BEHAVIOR)
```
Comment (DaveBirdsall) [inline core/sql/sqlcomp/CmpSeabaseDDLschema.cpp]: So, DROP SCHEMA will now work without CASCADE if SB_HISTOGRAMS exists? Nice change.

**Comment by DaveBirdsall** [comment]
Code looks good to me. Nice job.

File: core/sql/bin/SqlciErrors.txt (hunk 1 above)
Comment (robertamarton) [inline core/sql/bin/SqlciErrors.txt]: Will fix on a subsequent delivery.

**Pull request merged**